src/App.js

```diff
@@ -2,17 +2,18 @@ import './App.css';
 import initialCards from './cards-data';
 import Player from './components/Player';
 import CardList from './components/CardList';
-import { useState } from 'react';
+import { useContext, useState } from 'react';
 import ExecutePassButton from './components/ExecutePassButton';
+import { UserContext } from './context/CardContext.js';
 
 function App() {
   const [deck, setDeck] = useState(initialCards);
   const [playerOneHand, setPlayerOneHand] = useState([]);
-  const [selectedCard, setSelectedCard] = useState();
   const [playerTwoHand, setPlayerTwoHand] = useState([]);
   const [playerThreeHand, setPlayerThreeHand] = useState([]);
-  const [from, setFrom] = useState('deck');
-  const [to, setTo] = useState(1);
+  const { to } = useContext(UserContext);
+  const { from, setFrom } = useContext(UserContext);
+  const { selectedCard, setSelectedCard } = useContext(UserContext);
```

Comment:
you're using three lines to import the context but they're all coming from the same place — this would look much better as ```const { to, from, setFrom, selectedCard, setSelectedCard } = useContext(UserContext);```